packages/contracts/test/userStateTransition.test.ts

```diff
@@ -634,11 +638,12 @@ describe('User State Transition', function () {
 
         const users = 3
         const epochs = 3
-        let _stateTree = {}
+        let stateTreeMap = {}
```

Comment:
Does this need to be a map or can it just be 2 separate variables?

Reply:
Fix it here [a3aa6ce](https://github.com/Unirep/Unirep/pull/243/commits/a3aa6ce14e3b3c9a75963f15f108b05c2dca3b4d)